packages/config/src/lib/internal/resolver.spec.ts

```
@@ -0,0 +1,287 @@
+
+    const actual = resolve({ ...params, config });
+
+    expect(actual).toEqual({
+      database: {
+        host: "env-host",
+        port: "5434",
+      },
+      holidays: ["2024-01-01"],
+    });
+
+    delete process.env["HOLIDAYS"];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)